app/views/alert_mailer/daily_alert.text.haml

```diff
@@ -19,4 +19,4 @@ Visit #{notify_link('https://teaching-vacancies.service.gov.uk', t('app.title'))
 
 \---
 
-= t('subscriptions.email.unsubscribe_text_html', link: unsubscribe_link(@unsubscribe_token))
+= t('subscriptions.email.unsubscribe_text_html', unsubscribe_link: unsubscribe_link(@unsubscribe_token))
```

Comment:
This should probably also be `@subscription.token`, then it could be removed from the mailer.